src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: Props) => {
+  let selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  if (!selectedTab) {
+    selectedTab = tabs[0];
+  }
+
+  return (
+    <>
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === selectedTab?.id ? 'is-active' : ''}
```

Comment:
do not need to use here ternary operator with empty option, you can make it like this classname: condition